src/patterns/molecules/card/medium-card-cta.hbs

```diff
@@ -0,0 +1,12 @@
+---
+name: Medium Card with inline text CTA
+description: A medium card with image, headline, description, link, and inline text CTA
+order: 3
+notes: |
+    - Inline text CTAs should only be used in cards with descriptions
+links:
+    Card Layout Class: /patterns/templates/card-layout.html
+    Card Layout Demo: /demos/card-layout.html
+---
+
+{{#embed "patterns.molecules.card.small-card" class="mzp-c-card-medium mzp-has-aspect-3-2 mzp-has-video" cta="true" }}{{/embed}}
```

Comment:
There's nothing intrinsically unique about this card type itself, so i'm not sure if it warrants its own handle-bars template & example here? Seeing as this inline CTA is an optional feature that can be used on any card type, and not something unique to the medium sized card, it may just be worth adding a note about it in the documentation. Or as an example to one of the card types on the existing demo page perhaps?

Reply:
I've moved it into the medium card.